### PR TITLE
Feature/register opportunity

### PR DESCRIPTION
**Pull request by wdbasson**

### CODE REVIEW

File: src/web/youth/.rtx.toml
Comment (rblaine95) [inline src/web/youth/.rtx.toml:1]: This file isn't needed because there's a `.rtx.toml` in the root of the project

**Review by rblaine95 — CHANGES_REQUESTED** [review]

**Review by rblaine95 — CHANGES_REQUESTED** [review]

File: src/web/youth/package.json [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
     "@fontsource/open-sans": "^5.0.9",
-    "@hookform/resolvers": "^3.2.0",
+    "@hookform/resolvers": "^3.3.0",
     "@sentry/nextjs": "^7.64.0",
     "@t3-oss/env-nextjs": "^0.6.1",
     "@tanstack/react-query": "^4.33.0",
     "@types/react-modal": "^3.16.0",
     "axios": "^1.4.0",
-    "daisyui": "^3.5.1",
+    "cypress": "^12.17.4",
```
[the remark this answers is not in the document]
Comment (wdbasson) [reply src/web/youth/package.json]: Will remove for now...

**Review by rblaine95 — APPROVED** [review]
For what my review is worth, LGTM 👍

**Comment by rblaine95** [comment]
This is currently blocking #76 from progressing

**Comment by rblaine95** [comment]
@wdbasson @jasondicker can we close this PR?

**Pull request closed without merging**